### ISSUE TITLE
PlexAPI 4.0.0 removed support for py27 but still installs successfully on py27

### ISSUE THREAD

**Issue by andy-maier** [pkkid/python-plexapi#525]
**Describe the issue**

Installing PlexAPI 4.0.0 on Python 2.7 succceeds:

```
> python --version
Python 2.7.15
> pip install plexapi==4.0.0
> pip list | grep -i plexapi
PlexAPI            4.0.0
```

However, using it then fails:
```
> python -c "import plexapi"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\Andi\Envs\plexmediafixup\lib\site-packages\plexapi\__init__.py", line 9, in <module>
    from plexapi.utils import SecretsFilter
  File "C:\Users\Andi\Envs\plexmediafixup\lib\site-packages\plexapi\utils.py", line 10, in <module>
    from urllib.parse import quote
ImportError: No module named parse
```

`urllib.parse` is a new module in Python 3, where the functions of `urllib` were split into submodules.
The code that fails was added in PlexAPI 4.0.0. PlexAPI versions 3.6.0 and below run fine on Python 2.7.

So essentially, PlexAPI 4.0.0 removed support for Python 2.7, but it does not indicate that anywhere in its package on Pypi.

I am fully aware that Python 2.7 has reached its EOL. The issue is that PlexAPi 4.0.0 successfully installs on Python 2.7, and then fails.

**Code snipppets**

See above

**Expected behavior**

I am expecting that PlexAPI indicates in its package on Pypi which Python versions it supports, so that there is no discrepancy between successful installation and successful use.

**Enviroment (please complete the following information):**
 - OS: Windows
 - Plex version: latest
 - Python Version: 2.7.16
 - PlexAPI version: 4.0.0

**Additional context**

None

**Comment by Hellowlol** [comment]
Thanks for the report. I didn't do anything to prevent the package to be installed on python 2 and I don't intend to. I'll add some classifiers on the next pypi version so its listed on pypi

**Comment by andy-maier** [comment]
I have addressed the issue in PR #526. Let me know any comments.

**Comment by Hellowlol** [comment]
Awesome thanks.
